examples/push-file

```diff
@@ -3,11 +3,67 @@
 import argparse
 import json
 import logging
+import kobo.rpmlib
+import os
 
 from chexus import Client, BucketItem, TableItem
 
 LOG = logging.getLogger("push-file")
 
+def get_sigkey(rpmfile):
+    """
+    If the rpm is signed, return the sigkey.
+    rpmfile: path to an rpm
+    """
+    rpmhead = kobo.rpmlib.get_rpm_header(rpmfile)
+    rpmsig = kobo.rpmlib.get_keys_from_header(rpmhead)
+
+    # release, beta2, devel, old-release
+    valid_sigs = ["fd431d51", "f21541eb", "a5787476", "37017186"]
+
+    if rpmsig == None:
+        raise RuntimeError("Invalid RPM signature: %s is unsigned" % rpmfile)
+    if rpmsig.lower() not in valid_sigs:
+        raise RuntimeError("Invalid RPM signature: %s is signed with invalid key"  % (rpmfile, rpmsig.lower()))
```

Comment:
I think this is going to crash, because the number of %s in the string doesn't match the number of arguments you passed to %.

```suggestion
        raise RuntimeError("Invalid RPM signature: %s is signed with invalid key %s"  % (rpmfile, rpmsig.lower()))
```